plane/source/docs/quadplane-vtol-tuning.rst

```diff
@@ -0,0 +1,104 @@
+
+The most important parameters controlling stability are the Roll/Pitch/Yaw PIDS. For altitude control, the vertical position controller's parameters and Motor Thrust Scaling parameters, and for navigation/loiter the Loiter controllers's parameters
+
+Normally, it's best to start by tuning the Rate Roll/Pitch P in QSTABILIZE
+mode then move onto tuning altitude hold in QHOVER mode, then QLOITER
+(which often needs no tuning) and finally the waypoint navigation
+performance in Auto mode.
+
+
+Roll/Pitch tuning
+=================
+
+The ``Q_A_RAT_RLL_x`` and ``Q_A_RAT_PIT_x`` Roll/Pitch Rate parameters which convert the desired rotation rate into a motor output are the most important. 
+
+The :ref:`Q_A_ANG_RLL_P<Q_A_ANG_RLL_P>` and :ref:`Q_A_ANG_PIT_P<Q_A_ANG_PIT_P>` Roll/Pitch P converts the desired angle into a desired rotation rate which is then fed to the Rate controller.
+
+-  A higher value will make the QuadPlane more responsive to roll/pitch
+   inputs, a lower value will make it smoother
+-  If set too high, the QuadPlane will oscillate on the roll and/or pitch
+   axis
+-  If set too low the QuadPlane will become sluggish to inputs
+
+An objective view of the overall Roll and Pitch performance can be seen
+by graphing the :ref:`dataflash log's <common-downloading-and-analyzing-data-logs-in-mission-planner>` ATT message's DesRoll vs Roll and DesPit vs Pitch. The "Roll" (i.e.
+actual roll) should closely follow the "DesRoll" while in stabilized modes. Pitch should similarly closely follow DesPit.
+
+Alternatively you may wish to try tuning both the rate and angle
+parameters using the :ref:`QAUTOTUNE mode <qautotune-mode>`.
+
+Yaw Tuning
+==========
+
+The Angle Yaw and Rate Yaw parameters control the yaw response. With QuadPlanes, these often need tuning to get the desired YAW response, since configurations vary widely.
+
+Similar to roll and pitch, if either :ref:`Q_A_RAT_YAW_P<Q_A_RAT_YAW_P>` or :ref:`Q_A_ANG_YAW_P<Q_A_RAT_YAW_P>` is too high the QuadPlane's heading will oscillate. If they are too low, the QuadPlane may be unable to maintain it's heading.
+
+The :ref:`Q_A_ANG_YAW_P<Q_A_ANG_YAW_P>` is the gain on the error between the autopilots desired heading and actual heading which is fed into the Rate controller to demand a rotation rate. The :ref:`Q_A_RAT_YAW_P<Q_A_RAT_YAW_P>` is the gain applied to the difference between demanded rotation rate and actual.
+
+The :ref:`Q_YAW_RATE_MAX<Q_YAW_RATE_MAX>` parameter controls how quickly QuadPlane rotates based on a pilot’s yaw input in stabilized modes. 
+
+Altitude Tuning
+===============
+
+The QHOVER (altitude hold) related tuning parameters are related to the vertical position controller and the motor thrust scaling, which linearizes the throttle to motor thrust response to improve the position controllers response.
+
+TBD!
```

Comment:
needs some work :-)

Reply:
made me go and figure it out, didnt you?....took my best shot at it